server/src/Packages/Paypal/Services/PayPalClient.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace  Gernzy\Server\Packages\Paypal\Services;
+
+use PayPalCheckoutSdk\Core\PayPalHttpClient;
+use PayPalCheckoutSdk\Core\SandboxEnvironment;
+
+ini_set('error_reporting', E_ALL); // or error_reporting(E_ALL);
+ini_set('display_errors', '1');
+ini_set('display_startup_errors', '1');
+
+class PayPalClient
+{
+    /**
+     * Returns PayPal HTTP client instance with environment that has access
+     * credentials context. Use this instance to invoke PayPal APIs, provided the
+     * credentials have access.
+     */
+    public static function client()
+    {
+        return new PayPalHttpClient(self::environment());
+    }
+
+    /**
+     * Set up and return PayPal PHP SDK environment with PayPal access credentials.
+     * This sample uses SandboxEnvironment. In production, use LiveEnvironment.
+     */
+    public static function environment()
+    {
+        $clientId = config("api.paypal_client_id") ?: "";
+        $clientSecret = config("api.paypal_client_secret") ?: "";
+        return new SandboxEnvironment($clientId, $clientSecret);
```

Comment:
@AlexeiaW for switching between environments, should we make this an env or config variable?

Reply:
good idea, thanks for that, I implemented it